src/middlewares/isowner.js

```diff
@@ -15,11 +15,13 @@ export default async function isOwner(req, res, next) {
     const product = await products.findOne({ where: { id } });
     if (!product || product === null) return res.status(400).json({ message: "product doesn't exist" });
 
-    if (product.sellerId !== req.user.id) return res.status(401).json({ message: "you're not authorized" });
+    if (product.sellerId !== req.user.id && req.user.role !== 'admin') {
+      return res.status(401).json({ message: "you're not authorized" });
+    }
```

Comment:
Does this mean sellers can't see their products because they are not admins? Could you review the logic operator here?

Reply:
no i used the and operator which means that the only person who can delete,update and change availabilty of a product is the owner of the product where the sellerId match the one in the in product info and also the one who logged in who have role of admin